include/vsg/ui/PointerEvent.h

```diff
@@ -73,4 +73,15 @@ namespace vsg
             Inherit(in_window, in_time, in_x, in_y, in_buttonMask) {}
     };
 
+    VSG_type_name(vsg::WheelEvent);
+    class WheelEvent : public Inherit<PointerEvent, WheelEvent>
+    {
+    public:
+        WheelEvent(Window* in_window, time_point in_time, int32_t in_x, int32_t in_y, ButtonMask in_buttonMask, short in_delta) :
+            Inherit(in_window, in_time, in_x, in_y, in_buttonMask),
+            delta(in_delta) {}
+        
+        short delta;// multiple of WHEEL_DELTA (120)
```

Comment:
Where does the 120 come from?

Reply:
Hi，Robert. There is some information  about WM_MOUSEWHEEL in MSDN.

[MSDN](https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-mousewheel)


> #define WM_MOUSEWHEEL                   0x020A
> 
> Parameters
> wParam
> 
> The high-order word indicates the distance the wheel is rotated, expressed in multiples or divisions of WHEEL_DELTA, which is 120. A positive value indicates that the wheel was rotated forward, away from the user; a negative value indicates that the wheel was rotated backward, toward the user.

Reply:
That's quite bizarre interval. I think it would be best to just scale the delta to an interval of one prior to passing as an event, or for us to pass the interval with the even as well.  The aim should be to present a coherent set of values across platforms.  

Once I've got osg2vsg building against VulkanSDK 1.2.141.0 I'll come back to this issue and see what the other platforms do.  I'll also check what the OSG to see how it tries to avoid platform specific scaling.

Reply:
Yeah,you are right.